### ISSUE TITLE
Readme.md: Describe a day in the life of typical homeless person

### ISSUE THREAD

**Issue by tangospring** [codeforkansascity/street-medicine-kc#9]
One paragraph of text touching on important events: hygiene, food, clothing, drugs etc. 
Description for two people:
1. 45 y.o. veteran
2. Single woman with child.

**Comment by alfredosmkc** [comment]
Something to consider, within human services and specifically with agencies that work with homeless populations: there is no "typical homeless" individual.

**Comment by tangospring** [comment]
Yes, I fully expect that there are 3 to 5 such personas. We need to choose two the most common if we would like to make UI easy to use.
